packages/twenty-server/src/workspace/workspace-migration-runner/services/workspace-migration-enum.service.ts

```diff
@@ -135,20 +135,20 @@ export class WorkspaceMigrationEnumService {
     if (columnDefinition.defaultValue) {
       if (Array.isArray(columnDefinition.defaultValue)) {
         defaultValue = `ARRAY[${columnDefinition.defaultValue
-          .map((e) => `'${e}'`)
+          .map((e) => this.getStringifyValue(e))
```

Comment:
@thomtrp Did you test it again with custom objects ?
Because I was facing issue with just putting string into the query, but maybe it was only with TypeORM and not raw queries

Reply:
Not sure I test properly, how do you set an array as option in frontend?
I tried using [] but looks like it's red as a string: 
<img width="873" alt="Capture d’écran 2024-02-07 à 11 53 37" src="https://github.com/twentyhq/twenty/assets/22936103/e44a127e-879b-4ae4-b99b-86d572e81d97">

Reply:
For arrays the field type should be `MULTI_SELECT` and not `SELECT`, we must ensure before that `pg_graphql` is up to date and well supporting the arrays

Reply:
Discussed with you, this is not possible with our current version of pg_graphql. I removed the update for arrays since I can not test properly